components/example-notebook-servers/README.md

```diff
@@ -0,0 +1,139 @@
+# Example Notebook Servers
+
+**DISCLAIMER:** The Notebooks Working Group provides these Dockerfiles and
+their images as examples only, they are not tested with our CI/CD pipelines
+and are not certified to work in every situation. As such, issues related to
+these images will be dealt with in a best efforts approach. The Notebooks
+Working Group will work on officially supporting some of these Notebook
+Server images in the near future. If you do encounter a problem in one of
+these images, contributions and issue reports are greatly appreciated.
+
+## Introduction
+
+This directory contains Dockerfiles that are intended to show users how to
+create images for various types of Notebook Servers that can run on Kubeflow.
+There is a common base images that starts from Ubuntu 20.04, adds some
+dependencies and common utilities from APT, installs `kubectl`, creates the
+user account for the downstream images and adds the
+[S6-overlay](https://github.com/just-containers/s6-overlay) for the init system.
+From this base image, images for Jupyter, RStudio and Visual Studio Code
+(through [code-server]([https://github.com/cdr/code-server)) are created.
+From the Jupyter image, images are created that show users how to install
+TensorFlow and PyTorch both with and without support for NVIDIA GPUs.
+In turn, these images are extended with an opinionated set of common tools
+and packages to help new users to quickly get started with Kubeflow.
+
+**Lineage of the provided example Notebook Server Images:**
+
+![Flow Chart of example Notebook Servers](./kubeflow-example-notebook-servers.png)
```

Comment:
Could you upload an image with a white background? If you use GitHub with the Dark theme it's hard to see the arrow connections